src/main/java/tech/toparvion/analog/service/choice/ConditionalOnChoicesAutoReloadEnabled.java

```diff
@@ -0,0 +1,17 @@
+package tech.toparvion.analog.service.choice;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+
+import java.lang.annotation.*;
+
+/**
+ * @author Polyudov
+ * @since 0.14
```

Comment:
Here in AnaLog I used to add `v` prefix to all version numbers mentioned in the code:
```suggestion
 * @since v0.14
```
Let's keep this tiny part of consistence :)